src/services/AuthServices.js

```diff
@@ -6,23 +6,22 @@ export const setCurrentUser = currentUser => {
   api.setHeader('Authorization', currentUser.sessionToken);
   LocalStorageService.setSessionToken(currentUser.sessionToken);
 };
+
 export const getCurrentUser = () => {
   const currentSessionToken = LocalStorageService.getSessionToken();
-
   if (currentSessionToken) {
     api.setHeader('Authorization', currentSessionToken);
-
     return true;
   }
-
   return false;
 };
+
 export const removeCurrentUser = () => LocalStorageService.removeSessionToken();
 
-export const login = () =>
-  // TODO: Implement call to authentication API here
-  new Promise(resolve => {
-    setTimeout(() => {
-      resolve({ ok: true, data: { sessionToken: 'token' } });
-    }, 750); // eslint-disable-line no-magic-numbers
+export const loginToGithub = code => {
+  const authApi = api.setBaseURL(`${process.env.REACT_APP_AUTH_BASE_URL}?code=${code}`);
+  return authApi.post().then(response => {
+    setCurrentUser({ sessionToken: response });
+    window.history.replaceState({}, document.title, '/');
```

Comment:
Wouldn't this always this count as a "successful" login?
We are not checking the response.ok